<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegDialog</class>
 <widget class="QDialog" name="RegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_regUsr">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрировать пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_regStatus">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>42</y>
     <width>110</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_login1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>117</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_login1">
      <property name="text">
       <string>Введите логин:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_password1">
      <property name="text">
       <string>Введите пароль:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_password2">
      <property name="text">
       <string>Подтвердите пароль:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
